### DISCUSSION TITLE
Vepl.it - Repl.It project template for VLang

### DISCUSSION THREAD

**Discussion by SpencerDevv** [vlang/v#14127] [General]
![Vepl.it is cool](https://user-images.githubusercontent.com/94411511/164518896-02447170-6815-4302-b798-b657bb4644d3.png)

Hello everyone,

I have created a Repl.It template for VLang called Vepl.it! I know this will help me because our school chromebooks won't allow me to download VLang on it. 

There are 2 problems (that I know of): 
* There isn't any syntax highlighting 
* There is no Intellisense

the template link is [here!](https://replit.com/@SDevv/Veplit-VLang-Template)

Thank you for reading! :smiley:

**Comment by changrui** [comment]
@SpencerDevv  Thanks, I am using to do some simple tests.

**Comment by SpencerDevv** [comment]
Hello everyone,

I have updated Vepl.it to use Modules, so if you forked the template, can you re-fork it!

Also thank you @changrui for testing it out!

Sorry for the inconvenience, but Thanks for reading!
